ngrinder-frontend/src/js/main.js

```diff
@@ -104,6 +105,7 @@ new Vue({
     components: {
         Copyright,
         Navigator,
+        Messages,
```

Comment:
Deletable?

Reply:
It's necessary for this part
https://github.com/naver/ngrinder/pull/413/commits/1b5d0bd540018a2d64cd30ed52225583151bda98?file-filters%5B%5D=.js&file-filters%5B%5D=.vue#diff-d1f948ea1f38e5ed023c487a4447ab4cR16